assets/js/admin/orders/order-overview/models/state.js

```diff
@@ -73,49 +73,12 @@ export const State = Backbone.Model.extend(
 		getSubtotal() {
 			const { models: items } = this.get( 'items' );
 
-			// Use stored value if the record has already been created.
-			if ( false === this.get( 'isAdding' ) ) {
-				// Use the `discount` column for each OrderItem. OrderAdjustments do
-				// not store the per-item discount amount.
-				//
-				// Used when viewing a Refund to find the actual amount refunded per item.
-				//
-				// A $50.00 OrderItem that has a $12.50 discount applied by splitting two
-				// flat rate discounts of $5 and $10 would incorrectly try to apply $15.00
-				// discount if the attached OrderAdjustments were summed.
-				return items.reduce(
-					( amount, item ) => {
-						return amount += ( item.get( 'subtotal' ) - item.get( 'discount' ) );
-					},
-					0
-				);
-			}
-
-			// Add all subtotals.
-			const subtotal = items.reduce(
+			return items.reduce(
```

Comment:
I'm confused about why this is calculating the subtotal instead of using the order object `subtotal` similar to how `getTotal()` does (down at line 174).

Reply:
@robincornett Good point. With the new way it's calculated we can add parity here.